### PR TITLE
[KF-334] Add user action service client

### PR DESCRIPTION
**Pull request by 7eben**

### CODE REVIEW

File: kuflow-engine-client-common/src/main/java/com/kuflow/engine/client/common/resource/UserActionWorkflowRequestResource.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ * Copyright (c) 2021-present KuFlow S.L.
+ *
+ * All rights reserved.
+ */
+
+package com.kuflow.engine.client.common.resource;
+
+import java.util.UUID;
+
+public class UserActionWorkflowRequestResource extends AbstractResource {
```
Comment (jrpedrianes) [inline kuflow-engine-client-common/src/main/java/com/kuflow/engine/client/common/resource/UserActionWorkflowRequestResource.java:11]: Where do you use this class?

File: kuflow-engine-client-common/src/main/java/com/kuflow/engine/client/common/resource/UserActionWorkflowResponseResource.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2021-present KuFlow S.L.
+ *
+ * All rights reserved.
+ */
+
+package com.kuflow.engine.client.common.resource;
+
+public class UserActionWorkflowResponseResource extends AbstractResource {
```
Comment (jrpedrianes) [inline kuflow-engine-client-common/src/main/java/com/kuflow/engine/client/common/resource/UserActionWorkflowResponseResource.java:9]: Where do you use this class?

**Review by jrpedrianes — CHANGES_REQUESTED** [review]

File: kuflow-engine-client-common/src/main/java/com/kuflow/engine/client/common/resource/UserActionWorkflowRequestResource.java (hunk 1 above)
Comment (7eben) [reply kuflow-engine-client-common/src/main/java/com/kuflow/engine/client/common/resource/UserActionWorkflowRequestResource.java:11]: Is the input object of the action workflow method. I can reuse the WorkflowRequestResource and add the ActionvalueID, or create this new one. The object is a warpper of the ProcessId + UserActionId.

File: kuflow-engine-client-common/src/main/java/com/kuflow/engine/client/common/resource/UserActionWorkflowResponseResource.java (hunk 2 above)
Comment (7eben) [reply kuflow-engine-client-common/src/main/java/com/kuflow/engine/client/common/resource/UserActionWorkflowResponseResource.java:9]: Ídem but for the response.

```
@WorkflowInterface
public interface ActionSampleWorkflow {
    @WorkflowMethod(name = "MyActionWorkflow")
    UserActionWorkflowResponseResource runWorkflow(UserActionWorkflowRequestResource request);
}
```

File: kuflow-engine-client-common/src/main/java/com/kuflow/engine/client/common/resource/UserActionWorkflowRequestResource.java (hunk 1 above)
Comment (jrpedrianes) [reply kuflow-engine-client-common/src/main/java/com/kuflow/engine/client/common/resource/UserActionWorkflowRequestResource.java:11]: oook, perfect, I get it.

File: kuflow-engine-client-common/src/main/java/com/kuflow/engine/client/common/resource/UserActionWorkflowResponseResource.java (hunk 2 above)
Comment (jrpedrianes) [reply kuflow-engine-client-common/src/main/java/com/kuflow/engine/client/common/resource/UserActionWorkflowResponseResource.java:9]: oook, perfect, I get it.

**Review by jrpedrianes — APPROVED** [review]

**Pull request merged**